README.md

```diff
@@ -389,6 +389,215 @@ In the previous examples, Docker secrets files were used to specify admin creden
 
 Using Docker secrets, username and password information are secured when transmitting the sensitive data from Docker host to Docker containers. To prevent any attacks, the login information is not available as an environment variable. However, these values are stored in a text file and persisted in an in-memory file system inside the container. We recommend that you delete the Docker secrets information once the cluster is up and running.
 
+### How to use Docker Secrets with Docker Stack
+
+1. Run below to initialize swarm setup:
+
+```
+  $docker swarm init
+```
+2. Create secrets using the below commands:
+ - Create mldb_admin_username_v1.txt file to add mldb_admin_username_v1 secret for admin username using the following command:
+```
+  $docker secret create mldb_admin_username_v1 mldb_admin_username_v1.txt
+```
+  - Create mldb_admin_password_v1.txt file to add mldb_admin_password_v1 secret for admin password using the following command:
+```
+$docker secret create mldb_admin_password_v1 mldb_admin_password_v1.txt
+```
+3. Create marklogic-multi-centos.yml using below:
```

Comment:
Create the `marklogic-multi-centos.yml` using this command:

(shouldn't this be yaml and not yml?)

Reply:
Docker repo still has yml extension. We should keep it consistent for now and replace all extensions in one go.

Reply:
Sure, setting it back to yml.